.babelrc

```diff
@@ -10,8 +10,8 @@
           "last 5 Edge versions"
         ]
       },
-      "useBuiltins": true
+      "useBuiltIns": "entry"
```

Comment:
As Rob and I discovered, the `true` value here was really never an intended value, but did result in some stuff happening in preset-env 😄 . In any case, v7 of babel now validates that the values are correct, and we have to use either `false`, `entry`, or `usage` here.

Reply:
Just found out `usage` is an experimental value -- but theoretically that is what we want once it is stable.